### PR TITLE
fix lex errors

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@danroth27 please review

### CODE REVIEW

File: aspnet/security/data-protection/consumer-apis/password-hashing.rst [hunk 1]
```diff
@@ -1,16 +1,15 @@
 Password Hashing
```
Comment (danroth27) [inline aspnet/security/data-protection/consumer-apis/password-hashing.rst]: Did you mean to include these changes in this PR? They aren't lexer related, but they look ok.

**Comment by danroth27** [comment]
:shipit:

File: aspnet/security/data-protection/consumer-apis/password-hashing.rst (hunk 1 above)
Comment (Rick-Anderson) [inline aspnet/security/data-protection/consumer-apis/password-hashing.rst]: @danroth27  yes. It didn't take long to fix and it really needed word smithing.

**Comment by Rick-Anderson** [comment]
merged

**Pull request closed without merging**